modules/apps/dynamic-data-mapping/dynamic-data-mapping-taglib/src/main/resources/META-INF/resources/liferay-ddm.tld

```diff
@@ -73,6 +73,12 @@
 			<rtexprvalue>true</rtexprvalue>
 			<type>boolean</type>
 		</attribute>
+		<attribute>
+			<name>synchronousFormSubmission</name>
```

Comment:
@antonio-ortega , we cannot directly change .tld files. The correct process is:

1. Add the new input to [`liferay-ddm.xml`](https://github.com/liferay/liferay-portal/blob/master/modules/apps/dynamic-data-mapping/dynamic-data-mapping-taglib/src/main/resources/META-INF/resources/liferay-ddm.tld)
2. Run `gradlew buildTaglibs`

These steps will update the *.tld files propertly.